### PR TITLE
Add login feature

### PR DESCRIPTION
**Pull request by KarishmaSinghChauhan**

### CODE REVIEW

File: frontendservice/src/views/Login.vue [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        .catch(err => {
+          this.invalidInputAlert = "Could not sign in. " + err.message;
+        });
+      var user = firebase.auth().currentUser;
+    }
+  }
+};
+</script>
+
+<style scoped>
```
Comment (sdumka) [inline frontendservice/src/views/Login.vue:100]: Similar comment as the other PR on alphabetic sorting of attributes within each style group

File: frontendservice/src/views/Login.vue [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  margin-bottom: 1%;
+  width: 10%;
+  cursor: pointer;
+}
+p {
+  margin-bottom: 1%;
+  font-size: 0.8rem;
+  color: red;
+}
+p a {
```
Comment (sdumka) [inline frontendservice/src/views/Login.vue:124]: More elaborate naming if possible? Similarly in others

File: frontendservice/src/views/Login.vue [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    login() {
+      firebase
+        .auth()
+        .signInWithEmailAndPassword(this.email, this.password)
+        .then(user => {
+          this.$router.replace("shop-owner");
+        })
+        .catch(err => {
+          this.invalidInputAlert = "Could not sign in. " + err.message;
+        });
```
Comment (bavanya) [inline frontendservice/src/views/Login.vue:72]: can we have firebase.auth().signInWithEmailAndPassword() in same line?

File: frontendservice/src/views/Login.vue [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+      const provider = new firebase.auth.GoogleAuthProvider();
+      firebase
+        .auth()
+        .signInWithPopup(provider)
+        .then(result => {
+          this.$router.replace("verify");
+        })
+        .catch(err => {
+          this.invalidInputAlert = "Could not sign in. " + err.message;
+        });
```
Comment (bavanya) [inline frontendservice/src/views/Login.vue:93]: Here too similar to previous comment?

File: frontendservice/src/views/Login.vue (hunk 3 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/Login.vue:73]: At the dot operator, I've taken a newline. (Did the same in previous PR and it has been merged, so I thought that this way is acceptable.)

File: frontendservice/src/views/Login.vue (hunk 1 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/Login.vue:101]: Done.

File: frontendservice/src/views/Login.vue (hunk 2 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/Login.vue]: Done.

File: frontendservice/src/views/Login.vue (hunk 4 above)
Comment (KarishmaSinghChauhan) [reply frontendservice/src/views/Login.vue:94]: Same.

**Pull request merged**